packages/ui/src/MembersView/index.tsx

```diff
@@ -246,19 +246,55 @@ export default function MembersView({
     };
   }, []);
 
+  const validateEmail = (emailString: string): boolean => {
```

Comment:
Thanks a lot! Would you mind creating a new file and moving this function there?

```
packages/utilities/src/email.ts
```

For clarify, you could update the name to `isEmailValid` to make it more clear that a boolean is being returned.

You'll also need to update `packages/utilities/package.json` with a new line in `exports` and `typesVersions`.

Some basic specs would be appreciated too :). You can run specs in a package by navigating to the `packages/utilities` directory and running `npm run test`.